Fixed buggy AMD implementation

tHBp opened the pull request:
Implemented Universal Module Definition to support buggy existing AMD implementation.

adamschwartz commented:
@tHBp Thanks so much for your contribution. I would very much like to improve the library’s reusability by implementing UMD. However, this PR seems to have made some additional changes to the codebase, including changing the indentation from 2 spaces to 4, and adding `;` at the end of some lines. If you are willing to re-implement the PR such that the only change modifies the module definition, I’d be happy to accept the changes. Otherwise, I’d be happy to implement this change at some point myself. Thanks

tHBp commented:
I have implemented the suggested changes and updated the PR. If it is reasonable to you, can you also publish the same to npm once changes are merged? That'd be great. Thanks.

adamschwartz commented:
Should be all set https://www.npmjs.com/package/chrome-tabs
